servicenow/stackstate_checks/servicenow/servicenow.py

```diff
@@ -258,13 +265,16 @@ def _get_json(self, url, timeout, auth=None, verify=True):
             if status is AgentCheck.CRITICAL:
                 self.service_check(self.SERVICE_CHECK_NAME, status, tags=tags,
                                    message=msg)
-                raise Exception("Cannot connect to ServiceNow CMDB, please check your configuration.")
+                raise CheckException("Cannot connect to ServiceNow CMDB, please check your configuration.")
 
         if resp.encoding is None:
             resp.encoding = 'UTF8'
         try:
-            resp = yaml.safe_load(resp.text)
+            resp = yaml.safe_load(resp.text.encode("utf-8"))
```

Comment:
You've added this line. There must be an important reason for it. I've read nothing about this in the bug report? Perhaps add a test?

Reply:
I wrote the test for `_get_json` call with `utf-8` in response.